CONTRIBUTING.md

```diff
@@ -196,4 +195,10 @@ Before submitting a pull request, please make sure the following is done:
   - Example: `feature/my-awesome-feature` or `fix/annoying-bug`;
 - Build your changes;
 - If you’ve fixed a bug or added code that should be tested, **add tests**;
-- Ensure your commit is validated;
\ No newline at end of file
+- Ensure your commit is validated;
+
+### What happens next?
+
+After your pull request has been submitted, TotalCross code maintainers will acknowledge it as soon as possible. We aim to do it in three days tops.
+
```

Comment:
We can add a table to make it more graphical and _flashy_?

Reply:
That's a good idea! Something like this?

| Action | Action deadline |
|---|---|
| Pull request committed | -  |
| First reply | 3 days  | 
| Backlog prioritisation feedback | 7 days  |

Reply:
Yep, I think we are already in a good first version!

Reply:
Hey @nmarquesin **[this](https://github.com/TotalCross/totalcross/pull/144#discussion_r490436881)** need to be committed :sweat_smile:

Reply:
Ops! I got confused by the commit message error and forgot to do it properly. Should be fixed now 👍